newm-server/src/main/kotlin/io/newm/server/features/song/SongRoutes.kt

```diff
@@ -75,6 +76,15 @@ fun Routing.createSongRoutes() {
                         )
                     )
                 }
+                get("stream") {
```

Comment:
Would be great if you can add a Unit test for this route

Reply:
unit tests added